src/utils/sourcemaps.rs

```diff
@@ -247,6 +247,19 @@ fn is_remote_url(url: &str) -> bool {
     };
 }
 
+/// Return true iff url appears to be a URL path.
+/// Most often, a URL path will begin with `/`,
+/// particularly in the case of static asset collection and hosting,
+/// but such a path is very unlikely to exist in the local filesystem.
+fn is_url_path(url: &str) -> bool {
+    url.starts_with('/') && !Path::new(url).exists()
```

Comment:
As far as I know, `path.exists()` makes a system call. Can we avoid having validation only filesystem calls, and also achieve the same result of `is_url_path` purpose? `Path::new(url).is_absolute()` for example?

Reply:
The file non-existence check is an important part of `is_url_path`, as it prevents treating a local path as a remote URL path (without the `https://` part).